packages/components/src/components/chat-search/ContextStatus.vue

```diff
@@ -0,0 +1,208 @@
+<template>
+  <v-status-bar v-if="appmapStats && appmapStats.numAppMaps" level="success">
+    <template v-slot:header>
+      <div class="context__header">{{ appmapStats.numAppMaps }} AppMaps available</div>
+    </template>
+    <template v-slot:body>
+      <div class="context__table mb-0">
+        <div class="context__table__header">Navie's context:</div>
+        <div class="context__table__row">
+          <v-success-icon class="success-icon" data-cy="success-icon" />
+          <div>Runtime traces (AppMap recordings)</div>
+        </div>
+        <div class="context__table__row">
+          <v-success-icon class="success-icon" data-cy="success-icon" />
+          <div>Source code snippets</div>
+        </div>
+      </div>
+    </template>
+  </v-status-bar>
+
+  <v-status-bar v-else level="error">
+    <template v-slot:header>
+      <div class="context__header">No AppMaps available</div>
+    </template>
+    <template v-slot:body>
+      <div class="context__sub-header">
+        <div class="context__table">
+          <div class="context__table__header">Navie's context:</div>
+          <div class="context__table__row">
+            <v-fail-icon class="fail-icon" data-cy="fail-icon" />
+            <div>Runtime traces (AppMap recordings)</div>
+          </div>
+          <div class="context__table__row">
+            <v-fail-icon class="fail-icon" data-cy="fail-icon" />
+            <div>Source code snippets</div>
+          </div>
+        </div>
+        <p class="mt-0">
+          You can still ask Navie questions, but Navie does not have access to your project's data.
+        </p>
+        <template v-if="!appmapYmlPresent">
+          <p class="mb-0 mt-0" data-cy="status-no-config">
+            You need to
+            <a href="#" data-cy="install-guide" @click.prevent="openInstallInstructions">
+              finish setting up AppMap
+            </a>
+            and then
+            <a href="#" @click.prevent="openRecordInstructions" data-cy="record-guide">
+              create AppMap recordings
+            </a>
+            for Navie to know about your project.
+          </p>
+        </template>
+        <p
+          v-else-if="appmapStats && appmapStats.numAppMaps === 0"
+          class="mt-0 mb-0"
+          data-cy="status-no-data"
+        >
+          You need to
+          <a href="#" @click.prevent="openRecordInstructions" data-cy="record-guide">
+            record runtime data
+          </a>
+          for Navie to know about your project.
```

Comment:
Two options seem useful here:

1) A button or link to return to the recording instructions.
2) A sample prompt to ask Navie how to make AppMaps (let her take it from there).